### DISCUSSION TITLE
XCT Develop Branch Future

### DISCUSSION THREAD

**Discussion by GUOLDEV** [xamarin/XamarinCommunityToolkit#1866] [Q&A] [answered]
Is there a reason that develop branch is not being used for around 4 months? Are we going to use the main branch for now?

**Comment by bijington** [comment, marked as answer]
> Is there a reason that develop branch is not being used for around 4 months? Are we going to use the main branch for now?

We are only planning/accepting bug fixes on the toolkit so `main` is the right target there. `develop` was there to allow us to add functionality while still applying fixes but that is overkill now.

**Reply by GUOLDEV** [reply]
Thanks @bijington for the clarification 👍
